networkit/test/test_vizbridges.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+import networkit as nk
+from networkit import vizbridges
+import unittest
+from collections import defaultdict
+
+
+class TestVizbridges(unittest.TestCase):
+    def getSmallGraph(self, weighted=False, directed=False):
+        G = nk.Graph(4, weighted, directed)
+        G.addEdge(0, 1, 1.0)
+        G.addEdge(0, 2, 2.0)
+        G.addEdge(3, 1, 4.0)
+        G.addEdge(3, 2, 5.0)
+        G.addEdge(1, 2, 3.0)
+
+        return G
+
+    def testVizNodeScores(self):
+        for dim, directed, weighted in zip(
+            vizbridges.Dimension, [True, False], [True, False]
+        ):
+            with self.subTest(dim=dim, directed=directed, weighted=weighted):
+                G = self.getSmallGraph(weighted, directed)
+                vizbridges.widgetFromGraph(
+                    G,
+                    dimension=dim,
+                    nodeScores=list(range(G.numberOfNodes())),
+                )
+
+    def testVizNodePartition(self):
+        for dim, directed, weighted in zip(
+            vizbridges.Dimension, [True, False], [True, False]
+        ):
+            with self.subTest(dim=dim, directed=directed, weighted=weighted):
+                G = self.getSmallGraph(weighted, directed)               
+                partition = nk.Partition(G.numberOfNodes())
+                partition.allToSingletons()
+                vizbridges.widgetFromGraph(
+                    G,
+                    dimension=dim,
+                    nodePartition=partition,
+                )
+
+    def testVizNodePalette(self):
+        for dim, directed, weighted in zip(
+            vizbridges.Dimension, [True, False], [True, False]
+        ):
+            with self.subTest(dim=dim, directed=directed, weighted=weighted):
+                G = self.getSmallGraph(weighted, directed)
+                vizbridges.widgetFromGraph(
+                    G,
+                    dimension=dim,
+                    nodeScores=list(range(G.numberOfNodes())),
+                    nodePalette=[(i / 10, i / 10, i / 10) for i in range(4)],
+                )
+
+    def testVizShowIds(self):
+        for dim, directed, weighted in zip(
+            vizbridges.Dimension, [True, False], [True, False]
+        ):
+            with self.subTest(dim=dim, directed=directed, weighted=weighted):
+                G = self.getSmallGraph(weighted, directed)
+                vizbridges.widgetFromGraph(
+                    G,
+                    dimension=dim,
+                    nodeScores=list(range(G.numberOfNodes())),
+                    showIds=False,
```

Comment:
currently, `showIds=True` is the default (and tested implicitly in other tests), but we should test both `True` and `False` in case this changes in the future